### PR TITLE
Schema: Add APIKey core type

### PR DESCRIPTION
**Pull request by gamab**
**What is this feature?**

Adding cue schema to APIKeys.

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer**:

@IevaVasiljeva @eleijonmarck, as we discussed, I'm not sure if we are going to merge that one as we want to sunset APIKeys.
Wdyt?

### CODE REVIEW

File: packages/grafana-schema/src/index.ts [hunk 1]
```diff
@@ -5,3 +5,5 @@
  */
 export * from './veneer/common.types';
 export * from './index.gen';
+// Re-export duplicate types explicitly
+export type { OrgRole } from './index.gen';
```
Comment (gamab) [inline packages/grafana-schema/src/index.ts:9]: @Clarity-89 used your trick :+1:

**Review by Clarity-89 — APPROVED** [review]
🎉

**Review by joanlopez — APPROVED** [review]
LGTM! 🚀

**Review by IevaVasiljeva — APPROVED** [review]
LGTM, nice work Gabriel! I've got no objections merging it if you've already done the work 😄

**Review by mckn — APPROVED** [review]
LGTM!

**Pull request closed without merging**